doc/tutorial/get_started.md

```diff
@@ -210,7 +211,11 @@ In addition, you must configure the network interfaces so they can be used by Mi
 
 Complete the following steps on each VM (`micro1`, `micro2`, `micro3`, and `micro4`):
 
-1. Access the shell in the VM.
+   ```{tip}
+   Open four terminals, one on each micro vm, to run the commands in parallel on each VM. (tmux panel multiplexing can help here ;))
```

Comment:
```suggestion
   You can run the following commands in parallel on each VM. We recommend that you open three additional terminals, so that you have a terminal for each VM.
   ```
```